src/Microsoft.Fx.Portability.MetadataReader/Exceptions/SystemObjectNotFoundException.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.Fx.Portability.Analyzer.Resources;
+using Microsoft.Fx.Portability.ObjectModel;
+using System.Collections.Generic;
+using System.Globalization;
+
+namespace Microsoft.Fx.Portability.Analyzer.Exceptions
+{
+    /// <summary>
+    /// Exception thrown when assembly containing <see cref="System.Object"/>
+    /// cannot be found.
+    /// </summary>
+    public class SystemObjectNotFoundException : PortabilityAnalyzerException
+    {
+        public SystemObjectNotFoundException(IEnumerable<AssemblyReferenceInformation> assemblies)
```

Comment:
may be better to persist the list of items and override the message property

Reply:
kk will fix